app/addons/documents/tests/componentsSpec.react.jsx

```diff
@@ -0,0 +1,199 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+define([
+  'api',
+  'addons/documents/components.react',
+  'addons/documents/stores',
+  'addons/documents/actions',
+  'addons/documents/resources',
+  'testUtils',
+  "react"
+], function (FauxtonAPI, Views, Stores, Actions, Documents, utils, React) {
+  FauxtonAPI.router = new FauxtonAPI.Router([]);
+
+  var expect = utils.chai.expect;
+  var TestUtils = React.addons.TestUtils;
+
+  var resetStore = function (designDoc) {
+    var designDocs = new Documents.AllDocs([designDoc], {
+      params: { limit: 10 },
+      database: {
+        safeID: function () { return 'id';}
+      }
+    });
+
+    Actions.editIndex({
+      newView: false,
+      viewName: 'test-view',
+      designDocs: designDocs,
+      designDocId: designDoc._id
+    });
+  };
+
+  describe('View editor', function () {
+
+    describe('Toggle button', function () {
```

Comment:
Ah, I guess it is the naming: `componentsSpec.react.jsx`

in our test helper in our gruntfile we define a testfilename as:

```js

    mochaSetup: {
      default: {
        files: { src: helper.watchFiles(['[Ss]pec.js'], ['./app/addons/**/*[Ss]pec.js'])},
```